metrics/prometheus.go

```diff
@@ -134,15 +134,99 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 		opts:            opts,
 	}
 	if includedMetrics.Has(container.CpuUsageMetrics) {
+		if includedMetrics.Has(container.PerCpuUsageMetrics) {
+			c.containerMetrics = append(c.containerMetrics, []containerMetric{
+				{
+					name:        "container_cpu_user_seconds_total",
+					help:        "Container time spent in user mode",
+					valueType:   prometheus.CounterValue,
+					extraLabels: []string{"cpu", "namespace"},
+					getValues: func(s *info.ContainerStats) metricValues {
+						values := make(metricValues, 0, len(s.Cpu.Usage.PerCpuUser))
+						for i, value := range s.Cpu.Usage.PerCpuUser {
+							values = append(values, metricValue{
+								value:     float64(value) / float64(time.Second),
+								labels:    []string{strconv.Itoa(i), "user"},
+								timestamp: s.Timestamp,
+							})
+						}
+						return values
+					},
+				}, {
+					name:        "container_cpu_kernel_seconds_total",
```

Comment:
I believe a metric for kernel time already exists: container_cpu_system_seconds_total.  Please use the system one instead of introducing a new metric.